go-controller/pkg/crd/egressfirewall/v1/apis/clientset/versioned/clientset.go

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
It would be better to be conformant with the directory structure similar to the sample controller at https://github.com/kubernetes/sample-controller/tree/master/pkg/generated where all the generated files are grouped under generated folder

Reply:
We can reorganize this after the fact, our egressIP has this same issue too